sections/email-signup-banner.liquid

```diff
@@ -0,0 +1,281 @@
+{{ 'section-image-banner.css' | asset_url | stylesheet_tag }}
+{{ 'component-newsletter.css' | asset_url | stylesheet_tag }}
+{{ 'newsletter-section.css' | asset_url | stylesheet_tag }}
+{{ 'section-email-signup-banner.css' | asset_url | stylesheet_tag }}
+{{ 'component-rte.css' | asset_url | stylesheet_tag }}
+
+{%- style -%}
+  #Banner-{{ section.id }}::after {
+    opacity: {{ section.settings.image_overlay_opacity | divided_by: 100.0 }};
+  }
+  {%- if section.settings.adapt_height_image and section.settings.image != blank -%}
```

Comment:
This isn't working as it's not checking the right setting I think:
```suggestion
  {%- if section.settings.image_height ==  `adapt-image` and section.settings.image != blank -%}
```

Reply:
![](https://screenshot.click/21-10-qwd81-1020w.png)

Reply:
Good catch , Ludo!